pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -415,6 +415,16 @@ message CommandConsumerStatsResponse {
         optional uint64 msgBacklog                  = 15;
 }
 
+message CommandGetLastMessageId {
```

Comment:
since we are introducing a new command, can you bump the "ProtocolVersion"?

Reply:
Thanks, will add it.

Reply:
bumped proto version to v11

Reply:
Thanks. will add the checking